src/Agents/Dhadgar.Agent.Windows/Installer/README.md

```
@@ -0,0 +1,217 @@
+```
+
+## Output Location
+
+MSI files are output to:
+```
+bin\{Configuration}\{Platform}\en-US\MeridianConsoleAgent-{Version}-{Platform}.msi
+```
+
+## Installation
+
+### Interactive Install
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)